### PR TITLE
Unique review

### PR DESCRIPTION
**Pull request by clemieux55**

### CODE REVIEW

**Comment by jzgit** [comment]
add this to a migration
add_index :reviews, [:user_id, :movie_id], unique: true

add this to the review.rb
validates_uniqueness_of :user, scoped_to: movie

**Comment by x3igh7** [comment]
sutff should be fixed and ready for a merge

**Comment by x3igh7** [comment]
we should probably merge this to master soon - we're very far ahead on this branch atm.

andddd we have a lot of conflicts now

**Comment by wannabefro** [comment]
JASONNNNNN merge this thing!!!

File: spec/features/new_review_spec.rb [hunk 1]
```diff
@@ -9,19 +9,19 @@
   describe "if a user is signed in" do  
     before do
       visit new_user_session_path
-
+      save_and_open_page
```
Comment (jzgit) [inline spec/features/new_review_spec.rb]: remove this!

File: app/models/movie.rb [hunk 2]
```diff
@@ -6,6 +6,13 @@ class Movie < ActiveRecord::Base
   validates_presence_of :title, :year, :message => "Invalid Criteria"
   validates :year, :inclusion => 1875..(Time.now.year + 1)
 
+  def unique_review?(current_user)
```
Comment (jzgit) [inline app/models/movie.rb:10]: usually if your have a conditional that is just returning true/false you can simplify the method

```
def unique_review?(user)
  reviews.where(:user_id => user.id).empty?
end
```

**Comment by jzgit** [comment]
Go ahead and merge after making those last two suggested changes - good job!

**Pull request merged**